### PR TITLE
(Maintenance) Random values

### PR DESCRIPTION
**Pull request by guykogus**
🚀
This is more of a maintenance PR than anything. Just wanted to do some cleaning of usage of the random values/elements. I think this is a bit cleaner (and probably more performant in the case of the random strings).

I also updated the version just so that it points to the current version (it should be 1 ahead of the latest tag, right?).

## Checklist
- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [ ] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Comment by guykogus** [comment]
Do I need a changelog entry for this?

**Comment by LucianoPAlmeida** [comment]
@guykogus Since is just implementation details I don't think is required :))

File: Sources/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.4.0</string>
+	<string>4.7.0</string>
```
Comment (LucianoPAlmeida) [inline Sources/Info.plist]: I think we are in version 4.6.0

Comment (guykogus) [reply Sources/Info.plist]: 4.6.0 has already been released, so we're working on 4.7.0

Comment (guykogus) [reply Sources/Info.plist]: You never know, considering that Swift 5 may be out in not too long... We'll cross that bridge when we get to it 😄

**Pull request merged**